js/client-battle-tooltips.js

```diff
@@ -837,6 +837,11 @@ var BattleTooltips = (function () {
 		}
 		if (!accuracy || accuracy === true) return '&mdash;';
 		if (ability === 'No Guard') return '&mdash; (Boosted by No Guard)';
+		if (move.ohko) {
+			if (this.battle.gen === 1)
+				return accuracy + '% (Will fail if target\'s speed is higher)';
```

Comment:
The linter is extremely relaxed for the client, as its style is fairly more diverse than server's.

But this is one of the things that aren't seen elsewhere in the client...